cmd/shim/main.go

```diff
@@ -578,8 +608,15 @@ func runWithNesting(ctx context.Context, cmd *exec.Cmd) error {
 	var cmdErr error
 	engineErr = engine.Start(ctx, engineConf, func(ctx context.Context, r *router.Router) error {
 		go http.Serve(l, r) //nolint:gosec
-		cmdErr = cmd.Run()
-		return cmdErr
+		cmdErr = cmd.Start()
+		if cmdErr != nil {
+			return cmdErr
+		}
```

Comment:
Don't we need a `pipeWg.Wait` call before the cmd.Wait here too?

Reply:
@sipsma My bad, just added it!